cgc/triclustering.py

```diff
@@ -64,11 +64,29 @@ class Triclustering(object):
     :type col_clusters_init: numpy.ndarray or array_like, optional
     :param bnd_clusters_init: Initial band cluster assignment.
     :type bnd_clusters_init: numpy.ndarray or array_like, optional
+
+    :Example:
+
+    >>> import numpy as np
+    >>> Z = np.random.randint(100, size=(6, 10, 8)).astype('float64')
```

Comment:
Same as above, let's try to avoid using zeros:
```suggestion
    >>> Z = np.random.randint(1, 100, size=(6, 10, 8)).astype('float64')
```